otwcode/otwarchive
# AO3-6492 Update text for abuse report page and confirmation email

# Pull Request Checklist

  as the first thing in your pull request title (e.g. AO3-1234 Fix thing)
  until they are reviewed and merged before creating new pull requests.

## Issue

This PR is for the [Jira ticket AO3-6492:](https://otwarchive.atlassian.net/browse/AO3-6492) Improvement - Update text for abuse report page and confirmation email.

## Purpose
This pull request features text updates to the /abuse_report/new page, which is where users can submit a report of actions that may violate the terms of service, harassment, DMCA claims, and so forth.

This update updates the description and success message text of that form, as well as adds clarification text to the summary field, and moves the link field higher in the form.

The HTML and text emails for confirmation of a successful form submission for this page have also been updated with new text.

## Testing Instructions

This is my first PR to this project, as well as my first experience with Ruby/Ruby on Rails. I can check the text updates locally, but I wasn't sure how to test the confirmation emails. Any guidance on how to do so would be appreciated.

Similarly, if any code of mine is not up to the standards, please point it out. RoR looks very foreign to me, but I tried my best to follow the Coding Standards and mimic the styles I noted in other files. (And apologies for the double PR -- I fudged my fork).

I have a Jira account (forceofcalm@gmail.com) but do not have permissions to update the ticket status, nor to comment on tickets.

## Credit

calm they/them
